app-scroll-effects/app-scroll-effects-behavior.html

```diff
@@ -214,29 +215,28 @@
     },
 
     /**
-     * Creates an effect object from an effect name and configuration that can be used to run
+     * Creates an effect object from an effect's name that can be used to run
      * effects programmatically.
      *
      * @method createEffect
-     * @param {string} effectName The name of an effect defined in `Polymer.AppLayout.scrollEffects`
-     * @param {Object=} effectConfig The effect config object if the effect accepts config
-     *   values. (Optional)
-     * @return {Object} The effect object. This object has the following interface:
+     * @param {string} effectName The effect's name registered via `Polymer.AppLayout.registerEffect`.
+     * @param {Object=} effectConfig The effect config object. (Optional)
+     * @return {Object} An effect object with the following functions:
      *
-     *  * `obj.setUp()`, Sets up the requirements for the effect.
+     *  * `effect.setUp()`, Sets up the requirements for the effect.
```

Comment:
hmm. This isn't the `effectConfig` though. It's the value returned from `createEffect`.